gnovm/pkg/gnolang/gonative.go

```diff
@@ -300,12 +318,12 @@ func go2GnoValue(alloc *Allocator, rv reflect.Value) (tv TypedValue) {
 			rv = rv.Elem()
 		}
 	}
-	if rv.Type().PkgPath() != "" {
-		rt := rv.Type()
-		tv.T = alloc.NewType(&NativeType{Type: rt})
-		tv.V = alloc.NewNative(rv)
-		return
-	}
+	//if rv.Type().PkgPath() != "" {
+	//	rt := rv.Type()
+	//	tv.T = alloc.NewType(&NativeType{Type: rt})
+	//	tv.V = alloc.NewNative(rv)
+	//	return
+	//}
```

Comment:
I'm not really sure about removing this could you double check ? 
@thehowl

Reply:
Should panic, in our usages we never support declared types